src/test/test-fs-util.c

```diff
@@ -231,13 +232,18 @@ static void test_readlink_and_make_absolute(void) {
         free(r);
         assert_se(unlink(name_alias) >= 0);
 
+        assert_se(pwd = get_current_dir_name());
+
         assert_se(chdir(tempdir) >= 0);
         assert_se(symlink(name2, name_alias) >= 0);
         assert_se(readlink_and_make_absolute(name_alias, &r) >= 0);
         assert_se(streq(r, name));
         free(r);
         assert_se(unlink(name_alias) >= 0);
 
+        assert_se(chdir(pwd) >= 0);
+        free(pwd);
```

Comment:
this line should go now, otherwise we'll free pwd twice!

Reply:
Oh. Thanks.